content/Services/RHOBS/tenant-groups/README.md

```diff
@@ -0,0 +1,5 @@
+# Tenant Groups
+
+From business perspective we can divide RHOBS into a few main "Tenant Groups" that can hold multiple soft tenants.
```

Comment:
```suggestion
From a business perspective we can divide RHOBS into a few main "Tenant Groups" that can hold multiple soft tenants.
```

Reply:
And what are the tenant groups? :thinking:

Reply:
Two instance: Telemeter and MST. How can we name it differently?

Reply:
Are tenant groups an implementation-detail or just a term we're using for describing mutually exclusive tenants across different Observatorium deployments? If the former then keep, if the latter then we shouldn't really use them as it's tenant is an already overloaded term.